diff --git a/inc/core/class.dc.core.php b/inc/core/class.dc.core.php
index 317434b..2d1e6f8 100644
--- a/inc/core/class.dc.core.php
+++ b/inc/core/class.dc.core.php
@@ -1056,8 +1056,8 @@ final class dcCore
         }
 
         if ($cur->isField('user_pwd')) {
-            if (strlen($cur->user_pwd) < 6) {
-                throw new Exception(__('Password must contain at least 6 characters.'));
+            if (strlen($cur->user_pwd) < 1) {
+                throw new Exception(__('Password must contain at least 1 characters.'));
             }
             $cur->user_pwd = $this->auth->crypt($cur->user_pwd);
         }
